app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/client/RtcClient.kt

```diff
@@ -96,16 +109,27 @@ class RtcClient(
             mandatory.add(MediaConstraints.KeyValuePair(HANDSHAKE_VIDEO_OFFER, "true"))
             mandatory.add(MediaConstraints.KeyValuePair(HANDSHAKE_DTLS_SRTP_KEY_AGREEMENT, "true"))
         }
-        createConnection(factory ?: return)
+
+        setSpeakerphoneOn(context)
+
+        createConnection(factory ?: return, hasRecordAudioPermission)
+    }
+
+    private fun setSpeakerphoneOn(context: Context) {
+        (context.getSystemService(Context.AUDIO_SERVICE) as? AudioManager)?.run {
+            isSpeakerphoneOn = true
+        }
     }
 
-    private fun createConnection(factory: PeerConnectionFactory) {
+    private fun createConnection(factory: PeerConnectionFactory,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/9505a80fe42460a19170990c8c0b687614740b84/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/client/RtcClient.kt#L124"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>